ci-operator/step-registry/baremetal/lab/post/wipe/baremetal-lab-post-wipe-commands.sh

```diff
@@ -64,24 +81,20 @@ function reset_host() {
   local pdu_uri="${7:-}"
   local host="${bmc_forwarded_port##1[0-9]}"
   host="${host##0}"
+  if timeout -s 9 5m ssh "${SSHOPTS[@]}" "root@${AUX_HOST}" ipmicmd "${host}" power status | grep -iq "Error: Unable to establish IPMI" \
+  && [ -n "${pdu_uri}" ]; then
+    echo "Resetting ${host} pdu"
+    reset_pdu "${pdu_uri}"
+    sleep 90
+  fi
+
   timeout -s 9 10m ssh "${SSHOPTS[@]}" "root@${AUX_HOST}" prepare_host_for_boot "${host}" "pxe"
```

Comment:
```suggestion
  timeout -s 9 5m ssh "${SSHOPTS[@]}" "root@${AUX_HOST}" prepare_host_for_boot "${host}" "pxe"
```

Reply:
Left this unchanged.  `timeout` setting of 5 mins will be shorter than the 15  (7 mins 30 sec) retries to power off the host in `prepare_host_for_reboot`. Shorter timeout would effectively kill the function itself as explained in my earlier comments.